package-lock.json

```diff
@@ -14,6 +14,9 @@
         "packages/govuk-frontend-review",
         "docs/examples/*"
       ],
+      "dependencies": {
```

Comment:
Similar thing to my PR - if I run `npm install` locally, this gets wiped.

Reply:
🤦‍♂️

Reply:
✅ Sorted